project/ISSUE-TRIAGE.md

```diff
@@ -49,14 +49,37 @@ An issue can have multiple of the following labels.
 | area/api                  |
 | area/builder              |
 | area/cli                  |
+| area/distribution         |
 | area/kernel               |
+| area/logging              |
+| area/networking           |
+| area/plugins              |
+| area/project              |
 | area/runtime              |
+| area/security             |
+| area/security/apparmor    |
+| area/security/seccomp     |
+| area/security/selinux     |
+| area/security/trust       |
 | area/storage              |
 | area/storage/aufs         |
 | area/storage/btrfs        |
 | area/storage/devicemapper |
 | area/storage/overlay      |
 | area/storage/zfs          |
+| area/swarm                |
+| area/testing              |
```

Comment:
I'm thinking this is better than `kind/flakytest` (this is not really at the same level than bug or features).